Terminal.Gui/Resources/config.json

```diff
@@ -10,16 +10,16 @@
   // note that not all values here will be recreated (e.g. the Light and Dark themes and any property initialized
   // null).
   //
-  "$schema": "https://gui-cs.github.io/Terminal.Gui/schemas/tui-config-schema.json",
+  // TODO: V2 - Reference via http
+  "$schema": "../../docfx/schemas/tui-config-schema.json",
```

Comment:
Including it as a content file and just using a relative reference is probably a better idea anyway.

Because changes to the schema otherwise apply to versions that don't have code to go with them.

You could always make it a link to a tag or a permalink, but that also means changes that otherwise could be avoided by just including it. 🤷‍♂️ 

Could even _also_ have a commented out URL or a $comment element with the URL to go with it, to have it both ways for reference.